src/primitives.ts

```diff
@@ -1,59 +1,16 @@
 /**
- * Event names that can occur in the application.
- *
- * Subject to change as new features are added and new events are defined
- * and logged.
+ * General event names that can be reused across products.
  */
-export enum EventName {
-  APPROVE_TOKEN_TXN_SUBMITTED = 'Approve Token Transaction Submitted',
+export enum BasicEventName {
```

Comment:
Basic is a bit unclear. Any other ideas on names?

Reply:
Thoughts on `PrimitiveEventName`?

Reply:
`SharedEventName`?